rasa_core/policies/ensemble.py

```diff
@@ -123,7 +126,8 @@ def _persist_metadata(self, path, dump_flattened_stories=False):
             "python": ".".join([str(s) for s in sys.version_info[:3]]),
             "max_histories": self._max_histories(),
             "ensemble_name": self.__module__ + "." + self.__class__.__name__,
-            "policy_names": policy_names
+            "policy_names": policy_names,
+            "trained_at": self.date_trained
```

Comment:
can we just use the timestamp at persistence time as this happens right after actual training? in some sense the `persist()` method completes the training. in that case we wouldn't need the `self.date_trained` attribute

Reply:
I'm not sure. I agree that persist completes the training but having it inside the training function ensures that the time is logged as it is described (in case the script in question doesn't follow `.train()` with `.persist()` immediately after).

Reply:
You're right, but that's very much a special case. either way works, really